docs/event-types.mdx

```diff
@@ -8,7 +8,7 @@ import TabItem from "@theme/TabItem";
 Each message sent through Svix has an associated event type.
 Event types are identifiers denoting the type of the message being sent.
 
-Event types are just string, and can be anything. For example: `user.signup`, `invoice.paid` and `workflow.completed`.
+Event types are just a string, for example: `user.signup`, `invoice.paid` and `workflow.completed`.
```

Comment:
This is just a small fix, have a much more comprehensive change to this page on a separate branch (`liam/event-type-updates`). But I just wanted to fix the typo quickly and remove `can be anything` given that event types have the following pattern:

`^[a-zA-Z0-9\\-_.]+$`